kernel/src/net/socket/unix/addr.rs

```diff
@@ -30,6 +30,14 @@ impl UnixSocketAddr {
         Ok(bound)
     }
 
+    pub(super) fn already_bound(&self) -> Result<()> {
```

Comment:
The method means the addr can be bound to (unnamed) even the socket is already bound. How about rename the method as `bind_unnamed`?